docs/troubleshooting/troubleshooting.aad.md

```diff
@@ -0,0 +1,80 @@
+# Troubleshooting: AGIC pod stuck in not ready state
+
+## Illustration
+If AGIC pod is stuck in ready state, you must be seeing the following:
+```
+$ kubectl get pods
+
+NAME                                   READY   STATUS    RESTARTS   AGE
+<AGIC-POD-NAME>                        0/1     Running   0          19s
+mic-774b9c5d7b-z4z8p                   1/1     Running   1          15m
+mic-774b9c5d7b-zrdsm                   1/1     Running   1          15m
+nmi-pv8ch       
+```
+
+## Common causes:
+1. [Stuck at creating authorizer](#agic-is-stuck-at-creating-authorizer)
+1. [Stuck getting Application Gateway](#agic-is-stuck-getting-application-gateway)
+
+## AGIC is stuck at creating authorizer
+When the AGIC pod starts, in one of the steps, AGIC tries to get an AAD (Azure Active Directory) token for the identity assigned to it. This token is then used to perform updates on the Application gateway.
+This identity can be of two types:
+    1. User Assigned Identity
+    1. Service Principal
```

Comment:
2.Service Principal

Reply:
this is automatically converted to 2 during rendering.